source/codegen/metadata/nidcpower/functions.py

```diff
@@ -674,7 +674,7 @@
         'name': 'trigger',
         'direction': 'in',
         'type': 'ViInt32',
-        'enum': 'ExportSignal'
```

Comment:
Note: `ExportSignal` is a superset of `SendSoftwareEdgeTriggerType`, but nimi-python and our documentation references the values in `SendSoftwareEdgeTriggerType` for this param: https://github.com/ni/grpc-device/wiki/NI-DCPOWER-Trigger-And-Event-Functions#parameters-33